models/mrsegmentator/meta.json

```diff
@@ -0,0 +1,140 @@
+{
+    "id": "3b8e2f1d-4c5a-4b2a-8f3e-6a7d9e2c1b0d", 
+    "name": "mrsegmentator",
+    "title": "MRSegmentator",
+    "summary": {
+      "description": "MRSegmentator is an AI-based pipeline for the segmentation of 40 anatomical structures in MR images (with and without contrast).",
+      "inputs": [
+        {
+          "label": "Input Image",
+          "description": "The MR or CT scan of a patient (Thorax, Abdomen and Pelvis).",
+          "format": "DICOM",
+          "modality": "MRI|CT",
+          "bodypartexamined": "WHOLEBODY",
+          "slicethickness": "n/a",
+          "non-contrast": true,
+          "contrast": true
+        }
+      ],
+      "outputs": [
+        {
+          "type": "Segmentation",
+          "classes": [
+            "SPLEEN",
```

Comment:
I'll add `SPLEEN` to SegDB. I can lookup the codes, but feel free to add SCT code's if you already have them ;))

Reply:
@FJDorfner We're adding a quite broad label here, can you explain weather the model delineates the entire spine or the bone-structure of the spine?

Reply:
Hi @LennyN95, the model segments the bodies of the vertebrae individually. The disk between is not included in the label. This would be the yellow label in the attached Screenshot as a visual:) 

<img width="244" alt="Screenshot 2024-07-24 at 10 24 47" src="https://github.com/user-attachments/assets/fb93e3be-d004-41e2-b8b3-efafc9cf86b3">